pkg/apiserver/rest/storage_mocks.go

```diff
@@ -0,0 +1,126 @@
+package rest
```

Comment:
I would suggest renaming this file to `storage_mocks_test.go` since it's only used in tests. In the case that we wanted to export `legacyStoreMock` for other libraries to use, I would then suggest that we create a subdirectory called `mocks` so that we don't have mixed runtime and testing code.